gcsfs/core.py

```diff
@@ -1181,7 +1172,7 @@ def split_path(cls, path):
         else:
             return path.split("/", 1)
 
-    def validate_response(self, status, content, json, path):
+    def validate_response(self, status, content, json, path, headers):
```

Comment:
Perhaps have headers=None, so that you don't need to change the call signature elsewhere

Reply:
Thanks @martindurant; should be all set now.